internal/otelcollector/config/config_base.go

```diff
@@ -1,4 +1,4 @@
-package common
+package config
 
 type BaseConfig struct {
```

Comment:
I wonder if we can drop the **Config** suffix in this type and make the name just **Base**.

It will be used as `config.Base` in the downstream packages instead of `config.BaseConfig`.